addons/attachment_indexation/__manifest__.py

```diff
@@ -8,7 +8,7 @@
 Attachments list and document indexation
 ========================================
 * Show attachment on the top of the forms
-* Document Indexation: odt
+* Document Indexation: odt, pdf, xlsx, docx
```

Comment:
Perhaps add a note about requiring the optional `pdfminer.six` library for successful PDF indexing, so users see it upfront?